lib/elixir/lib/enum.ex

```diff
@@ -1181,7 +1181,7 @@ defmodule Enum do
   Returns the biggest of the elements in the enumerable according
   to Erlang's term ordering.
 
-  If more than one elements compare equal, the first one that was found
+  If multiple elements are considered the biggest, the first one that was found
```

Comment:
Thanks for the feedback!

I tried to stay in the language of the documentation, which also uses biggest/smallest - if I'd make that change I would also change that. I'd like the change but wasn't yet ready to make it :)